CHANGES.md

```diff
@@ -1,3 +1,8 @@
+Airbase 104
+
+* Plugin updates:
+  - git-commit-id-plugin 4.0.0 (from 4.0.2) - downgraded due to https://bugs.eclipse.org/bugs/show_bug.cgi?id=564202
```

Comment:
Let's remove this bug link from the release notes and move it the commit message

Reply:
is there a link for git-commit-id issue? 
it would be more interesting as the maven plugin can workaround a big in the underlying lib, without the bug being fixed.